src/Dropdown.jsx

```diff
@@ -0,0 +1,170 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+class DisplayContainer extends React.Component {
+  constructor(...args){
+    super(...args);
+    this.handleSelection = this.handleSelection.bind(this);
+    this.handleClickOutside = this.handleClickOutside.bind(this);
+    this.setWrapperRef = this.setWrapperRef.bind(this);
+    this.clicked = this.clicked.bind(this);
+    this.state = {
+      displayValue: 'Good for Day',
+      clicked: false,
+    }
+  }
+
+  componentDidMount() {
+    document.addEventListener('mousedown', this.handleClickOutside);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('mousedown', this.handleClickOutside);
```

Comment:
Could consider adding any other cleanup tasks here (clear out rendered child components in a popup)